README.md

```diff
@@ -1 +1,155 @@
-# awesome-book
\ No newline at end of file
+<!-- @format -->
+
+# awesome-book<!-- @format -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 ["awesome book project of Microverse] <a name="about-project"></a>
+
+> This project is about adding books to the page and saving in local storage
+
+**[awesome book project of Microverse]** is a great project for beginners to learn how to build a simple use objects
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- **HTML**
+- **CSS**
+- **JS**
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Using Linters**
+- **objects for dynamic manipulation**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link] ()
```

Comment:
- [ ] [optional] @team great work on your documentation however I would recommend you remove the live demo section since you don't have a live demo currently, also adding all authors in the authors' section.